EVEStandard/Models/Wars.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EVEStandard.Models
+{
+    public class Wars : ModelBase<Wars>
+    {
+        /// <summary>Return a list of wars.</summary>
+        /// <value>Return a list of wars.</value>
+        [JsonProperty("get_wars_ok")]
+        public int[] GetWars { get; set; }
+    }
+
+    class WarId : ModelBase<WarId>
```

Comment:
By the model naming convention I've been trying to create, but haven't documented yet, this should be called WarInfo instead of WarId.

Reply:
Classes are internal by default so need to add public.